spec/factories/sponsors.rb

```diff
@@ -16,6 +16,11 @@
     branch { Branch.all.sample if sponsor_type.name == 'Individual' }
     organization { Organization.all.sample if sponsor_type.name == 'Organization' }
     payment_plan { Sponsor::PAYMENT_PLANS.sample }
+    address { "#{Faker::Address.street_name} #{Faker::Address.building_number.to_s}" }
+    email { Faker::Internet.email }
+    contact1 { Faker::PhoneNumber.phone_number }
+    contact2 { Faker::PhoneNumber.cell_phone }
+    additional_info { Faker::Lorem.sentence }
     agent
   end
 end
```

Comment:
I added new fields to the `:sponsor` factory to check their presence in the `_form` partial. These new fields are all optional in the model (no presence validation).
Should I just create a new `:full_sponsor` factory instead of editing `:sponsor`? (I think not)